dll/win32/imm32/imm.c

```diff
@@ -1017,13 +1018,331 @@ LRESULT WINAPI ImmEscapeW(
         return 0;
 }
 
+#ifdef __REACTOS__
```

Comment:
At this point we can better fork it and drop all ifdef?

Reply:
Forked and removed #ifdef's.